.github/CODEOWNERS

```
@@ -1 +1,2 @@
 *	@confluentinc/kafka-eng
+*	@confluentinc/ksql
```

```suggestion
*	@confluentinc/kafka-eng @confluentinc/ksql	
```

The syntax for this file can be confusing. Multiple owners for a given pattern go on the same line. If there are two equivalent entries, the last one wins